controllers/storagecluster/platform_detection.go

```diff
@@ -3,14 +3,19 @@ package storagecluster
 import (
 	"context"
 	"fmt"
-	"sync"
-
 	configv1 "github.com/openshift/api/config/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"strings"
+	"sync"
 )
 
+//IBMCloud secret name
```

Comment:
Please explain what `Cos` means here.

Reply:
I have made the suggested changes. Please could you take a look @jarrpa